erc20-load-test-tool/src/main/java/com/vmware/ethereum/service/WorkloadCommand.java

```diff
@@ -202,4 +211,56 @@ private void receiptProcessor(
       countDownLatch.countDown();
     }
   }
+
+  @SneakyThrows(InterruptedException.class)
+  private void retryReadCheck(BatchRequest retryBatchRequest, Instant startTime) {
+    BatchResponse receiptBatchResponse = null;
+    boolean success;
+    for (long retry = 1; retry <= 5; retry++) {
```

Comment:
Is this hard-coding of retries intentional?

Reply:
made configurable